### DISCUSSION TITLE
Chinese language support is needed

### DISCUSSION THREAD

**Discussion by AaronC-Alpha** [janstehno/cotw-companion#57] [Ideas]
As you can see, I am a Chinese speaking user. There are many players in China ,who all eager to have software as convenient as this. Therefore, we hope that it can provide a Chinese version.
Meanwhile, considering that developers may not have learned Chinese and other languages that may be needed. I suggest you can develop a translation function that corresponds the original text with content from other languages through a file. People can upload another files like this so that developers from other languages can participate in the development of the software. (Translated by _Baidu Fanyi_)

> Translated with DeepL.com

[标题] 需要更多的语言支持
[正文] 如你所见，我是一名说中文的用户。在中国有很多玩家游玩这个游戏，他们像我一样迫切地希望能够有像这个一样方便的软件。因此我们希望可以它提供中文版本。
同时，考虑到开发者可能没有学习过中文以及其他可能需要的语言。我建议您开发一个翻译功能，通过一个文件将原文与其他语言的内容对应，并允许添加新的语言文件，这样就可以让其他语言的开发者参与到软件的发展中

> Original

**Comment by janstehno** [comment]
Hello, so the translation to the Chinese language was suggested almost a year ago and this user wanted to help me create it. However after some time, he probably did not have much time anymore, so it was abandoned.

If you would like to help me with it and complete the translations, you can do so by opening the Google sheet [here]() and comment anything that has to be translated or is translated wrong. You can share the link so others can help too. Any kind of help will be appreciated.

**Reply by AaronC-Alpha** [reply]
I have received your reply. Due to restrictions in my region, I am unable to access Google Forms services stably. Therefore, I will try to create a Chinese translation in the offline form and contact you via email. In fact, I have already sent an email to the email you reserved in the app, and I would greatly appreciate it if you could receive it and be happy to reply. (Translated by _Baidu Fanyi_)

Original text:
您的回复我已收到。由于我所在地区的限制我不能稳定的访问谷歌表格服务。因此我会尝试在离线的表格中制作中文翻译并用邮件与你联系。事实上我已经向你在app中预留的邮箱发送了邮件，如果您可以收到并乐意回复的话我会十分感激。

**Comment by janstehno** [comment]
Added in 1.9.6
